WebRoot/h/search

```diff
@@ -5,13 +5,15 @@
 <%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
 <%@ taglib prefix="fn" uri="http://java.sun.com/jsp/jstl/functions" %>
 <%@ taglib prefix="fmt" uri="com.zimbra.i18n" %>
-
 <app:handleError>
     <zm:getMailbox var="mailbox"/>
     <c:choose>
     <c:when test="${not empty mailbox.prefs.locale}">
         <fmt:setLocale value='${mailbox.prefs.locale}' scope='request' />
     </c:when>
+    <c:when test="${not empty param.lang}">
```

Comment:
looks like this new code only runs during errors? Since it's within `<app:handleError>`. Though I may be misunderstanding what is happening here

Reply:
This code is simple updating Locale based on condition.

Reply:
this code should replace the `<c:when/>` below it, they are both setting the locale, so the second one is actually going to be the one that takes effect